extensions/wikia/GlobalNavigation/GlobalNavigationAccountNavigationController.class.php

```diff
@@ -144,8 +162,13 @@ private function setupPersonalUrls() {
 			}
 			$returnto = wfGetReturntoParam( $returnto );
 
-			$this->personalUrls[ 'login' ] = [ 'title' => wfMessage( 'login' )->text(), 'href' => Skin::makeSpecialUrl( 'UserLogin', $returnto ), 'class' => 'ajaxLogin table-cell' ];
-			$this->personalUrls[ 'register' ] = [ 'text' => wfMessage( 'oasis-signup' )->text(), 'href' => Skin::makeSpecialUrl( 'UserSignup' ), 'class' => 'ajaxRegister' ];
+			if ($this->enableNewModalAuth) {
+				$this->personalUrls['login'] = [ 'title' => wfMessage( 'global-navigation-sign-in' )->escaped(), 'href' => '/signin', 'class' => 'sign-in' ];
```

Comment:
aren't we missing ```redirect``` query param?

Maybe for URL generation we could use this helper https://github.com/Wikia/app/blob/dev/extensions/wikia/UserLogin/UserLoginHelper.class.php#L606 ?